h2o-core/src/test/java/water/rapids/SortTest.java

```diff
@@ -261,6 +261,24 @@ private static void TestSortOneColumn(String fileWithPath, int colIndex, boolean
     }
   }
 
+  @Test public void TestSortIntegersDescend() throws IOException {
```

Comment:
method should not start with a capital letter

Reply:
Yes, copied old tests.  Fixed them all.